samples/pollDiscoveryOccurrenceFinished.js

```diff
@@ -0,0 +1,54 @@
+// [START containeranalysis_poll_discovery_occurrence_finished]
+// Repeatedly query the Container Analysis API for the latest discovery occurrence until it is
+// either in a terminal state, or the timeout value has been exceeded
+const pollDiscoveryOccurrenceFinished = async (
+    projectId = 'your-project-id', // Your GCP Project ID
+    imageUrl = 'my-url.io/project/image', // Image to attach metadata to
+    timeoutSeconds = 'timeout-in-seconds', // The number of seconds to listen for the new Pub/Sub messages
+) => {
+    // Import the library and create a client
+    const grafeas = require('@google-cloud/grafeas');
+    const client = new grafeas.v1.GrafeasClient();
+
+    const formattedParent = client.projectPath(projectId);
+
+    let filter = `resourceUrl=\"${imageUrl}\" AND noteProjectId=\"goog-analysis\" AND noteId=\"PACKAGE_VULNERABILITY\"`;
+    // [END containeranalysis_poll_discovery_occurrence_finished]
+    filter = `kind = \"DISCOVERY\" AND resourceUrl = \"${imageUrl}\"`;
```

Comment:
You might want to leave a comment for future maintainers, since what we're doing is non-standard. This is what I wrote in Ruby:

>  The above filter isn't testable, since it looks for occurrences in a locked down project
>  Fall back to a more permissive filter for testing